arch/univariate/distribution.py

```diff
@@ -217,6 +217,57 @@ def cdf(self, resids, parameters=None):
         """
         pass
 
+    @abstractmethod
+    def moment(self, h, parameters=None):
+        """
+        Moment
+
+        Parameters
+        ----------
+        h : int
+            Order of moment to calculate
+        parameters : ndarray
+            Distribution parameters. Use ''None'' for parameterless
```

Comment:
You can use full width of 80 characters here. Should be 1 line I think.  

Remove '' around None.